solana/pyth2wormhole/client/src/main.rs

```diff
@@ -291,6 +294,16 @@ async fn handle_attest_daemon_mode(
     attestation_cfg: AttestationConfig,
     metrics_bind_addr: SocketAddr,
 ) -> Result<(), ErrBox> {
+    // Update healthcheck window size from config
+    {
+        let mut hc = HEALTHCHECK_STATE.lock().await;
+        hc.max_window_size = attestation_cfg.healthcheck_window_size as usize;
+
+        if hc.max_window_size == 0 {
```

Comment:
I think using None would be better than 0 to disable the check.

Reply:
I don't like that because explicit none is hard to represent in serde. Not specifying the value is not a good way to say "off" because then there's no good way to say "use default". I think I'll add an explicit on/off option for the healthcheck and forbid the 0 value (it no longer has a purpose if it doesn't have the special "off" meaning)